tests/jobscript/10-bad-syntax/suite.rc

```diff
@@ -0,0 +1,14 @@
+[cylc]
+    [[events]]
+        timeout = PT10S
+[scheduling]
+    [[dependencies]]
+        graph = foo:submit-failed => bar
```

Comment:
Does the suite die, because you have not got a list of tasks that are expected to fail?